### PR TITLE
feat(i18n): locales rework

### LINKED ISSUE #101: new i18n structure
- [x]  change fallback to en
- [x]  render faq in template
- [x]  storage na language chooser
- [ ]  DRY links as numbers into one file from faq 
  - Seems interpolation is quite hard https://github.com/dkfbasel/vuex-i18n/issues/48 && https://github.com/Gvade/vuex-i18n#interpolation
  - https://stackoverflow.com/questions/42649866/best-practice-for-html-tags-in-vue-i18n-translations
  - maybe wrap it into markdown? https://github.com/kazupon/vue-i18n/issues/330
  - dangerous umm https://github.com/kazupon/vue-i18n/issues/898
- [ ] Foldable answers, use https://buefy.org/documentation/collapse#accordion-effect
  - [ ] Visual like https://nftshowroom.com/faq foldable, got foldable cards https://github.com/kodadot/nft-gallery/issues/47

### PR DESCRIPTION
**Pull request by roiLeo**
In Draft should be reviewed.
This issue was a bit painful but now I'm a copy/paste master and fluent in all languages.
Let me know if we should keep .py script.

### Before submitting this PR, please make sure:
- [ ] Code builds clean without any errors or warnings (need help with last require and README warn)
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I didn't break any original functionality

### Optional
- [x] I've tested it on mobile and everything works

### PR type
- [x] Feature

### What's new? (maybe part of changelog)
- closes #101 
- #284 related

### Reworked locales
- [x] bn.json
- [x] cn.json
- [x] cz.json
- [x] de.json
- [x] en.json
- [x] es.json
- [x] fr.json
- [x] hi.json
- [x] it.json
- [x] jp.json
- [x] ko.json
- [x] nl.json
- [x] pl.json
- [x] pt.json
- [x] ru.json
- [x] sk.json
- [x] tu.json
- [x] ua.json
- [x] ur.json
- [x] vt.json

### CODE REVIEW

**Comment by yangwao** [comment]
> This issue was a bit painful but now I'm a copy/paste master and fluent in all languages.
> Let me know if we should keep .py script.

vous pourrez désormais parler plusieurs langues 😄

Okay I've checked for Slovak it's working https://deploy-preview-656--nftkodadot.netlify.app/rmrk/faq
Maybe remove <b> tags and leverage markdown bold using **  ?

For me all good, it's looking amazing and we can finally unfreeze translations!
Let me pass this to stewards to check their languages if anything critical is there and we can still fix it in new way others making PR to their language translations.

**Comment by vikiival** [comment]
> Let me know if we should keep .py script.

Yes please ❤️

**Comment by yangwao** [comment]
Oh, one last question @roiLeo 

What about adding that foldable effect? So whenever one searching for a question, one don't need to see all answers
It's the last point from https://github.com/kodadot/nft-gallery/issues/101 
- Foldable answers, use https://buefy.org/documentation/collapse#accordion-effect
 Visual like https://nftshowroom.com/faq foldable, got foldable cards

**Comment by vikiival** [comment]
> Oh, one last question @roiLeo
> 
> What about adding that foldable effect? So whenever one searching for a question, one don't need to see all answers
> It's the last point from #101
> 
> * Foldable answers, use https://buefy.org/documentation/collapse#accordion-effect
>   Visual like https://nftshowroom.com/faq foldable, got foldable cards

Let's make it a separated good first issue :)

**Comment by roiLeo** [comment]
@yangwao I didn't know if it was still relevant but I can propose something.

I still got one issue that I can't fix "Detected usage of the any type" on i18n.ts file.

**Comment by yangwao** [comment]
> @yangwao I didn't know if it was still relevant but I can propose something.

Yeah, I mean, rather wait and accept it once it's working, it will be huge upgrade for FAQs :)

> I still got one issue (Warning) that I can't fix "Detected usage of the any type" on i18n.ts file.

That sounds like typescript warning 🤔

**Comment by roiLeo** [comment]
I made some changes to include collapsible in faq.
By default all opened.
UI looks weird... Should I remove arrow ? or maybe change title width ?

### Screenshot
Screenshot 2021-08-30 at 14-51-26 KodaDot - Kusama NFT Market Explorer Low Carbon NFTs

**Comment by yangwao** [comment]
I mean looks good, I would 
- put chevron to the left (to not be random placed)
- different colour for chevron, maybe try $main? To know it's actionable.
- maybe smaller text for headline

All good for me!

image

File: src/components/rmrk/Faq-rework.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    </h2>
+    <p class="faq__box heading heading-is-2 text-bold is-flex-inline mb-6">
+      No time for reading? <br />
+      No problem. Check out <br />
+      <router-link :to="{ name: 'tutorials' }">
+        Our tutorials.
+      </router-link>
+    </p>
+    <div v-for="qa in faqQuestionsAnswers" v-bind:key="qa[0]" class="mb-5">
+      <b-collapse :open="true" :id="qa[0]">
```
Comment (vikiival) [inline src/components/rmrk/Faq-rework.vue:16]: `:open="false"`

**Review by vikiival — COMMENTED** [review]
Awesome work.

maybe rename src/components/rmrk/Faq-rework.vue into FAQ or so ? 

Awesome work.Awesome work.

**Comment by roiLeo** [comment]
### Screenshot
Screenshot 2021-08-31 at 09-04-04 KodaDot - Kusama NFT Market Explorer Low Carbon NFTs

I still have a small problem with scrolling to the anchor. (For example when you click on chevron)
Same with link like [/rmrk/faq#What-wallet-do-I-need-to-mint,-buy-and-collect?]() doesn't work when I don't have the right locale.

**Comment by yangwao** [comment]
I mean, we can flex addressable later stage...
right now to just dispatch it to live I see this one issue there that you lose headline text as it seems viewport starts under navbar if I see right?

All good, really like this going unfreeze translations!:)

https://user-images.githubusercontent.com/5887929/131492662-3a5f3e60-e5bb-4485-b762-4f8437410ddb.mov

**Comment by yangwao** [comment]
Ship it! 🚀

**Pull request merged**